src/auth/User.cc

```diff
@@ -23,14 +23,13 @@
 Auth::User::User(Auth::SchemeConfig *aConfig, const char *aRequestRealm) :
     auth_type(Auth::AUTH_UNKNOWN),
     config(aConfig),
-    ipcount(0),
     expiretime(0),
     credentials_state(Auth::Unchecked),
     username_(nullptr),
-    requestRealm_(aRequestRealm)
+    requestRealm_(aRequestRealm),
+    ipList(256*1024) // XXX: figure a more reasonable value than ~256KB cache per-username
```

Comment:
There was no limit before this PR, right? In PR description, please disclose this functionality change and discuss its expected effects.

Are there any other Squid functionality changes in this PR?